linters/jshint.py

```diff
@@ -0,0 +1,13 @@
+def options(ctx):
+    ctx.add_option('--jshint', action='store_true', help="Run JSHint on the JS files in the build")
```

Comment:
Why make jshint optional and defaulting to off? The app developer has already chosen to use jshint by loading the jshint tool; why require them to enable it twice?